javascript/bilingual_localization.js

```diff
@@ -123,7 +123,18 @@
         i18nRegex[key.slice(2)] = value
       } else if (regex_scope.test(key)) {
         // parse scope translations
-        const { scope, skey } = key.match(regex_scope).groups
+        let { scope, skey } = key.match(regex_scope).groups
```

Comment:
After adding support for CSS selectors, the regex pattern `regex_scope` at L119 needs to be changed. The old pattern  won't work when the scope has spaces. An example is `/^##(?<scope>[\S ]+)##(?<skey>\S+)$/`